lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -200,25 +200,28 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
         MSG
         @logger.summary.add_paragraph(ColorizedString.new(warning).yellow)
       end
-      StatsD.event("Deployment of #{@namespace} succeeded",
+      tags = statsd_tags + ["status:success"]
+      StatsD.client.event("Deployment of #{@namespace} succeeded",
         "Successfully deployed all #{@namespace} resources to #{@context}",
-        alert_type: "success", tags: statsd_tags << "status:success")
-      StatsD.distribution('all_resources.duration', StatsD.duration(start), tags: statsd_tags << "status:success")
```

Comment:
I think this was a (minor) bug. `<<` modifies the array, and then returns it. This means that every time it is called, the array grows by one element. In this case, we add the same tag twice, which doesn't really cause any issues in practice, but I figured I should fix this while I am at it.